modules/configuration-nix.nix

```diff
@@ -84,4 +84,32 @@ in {
     # this adds support for __int128_t and __uint128_t to language-c
     (fromUntil "0.9.1" "0.9.2" ../patches/languge-c-int128.patch)
   ];
+
+  packages.discount.components.library.libs = pkgs.lib.mkForce [ pkgs.discount ];
```

Comment:
Shouldn't these be in `hackage-quirks`?

Reply:
`hackage-quirks` can override `cabal configure` settings and add modules, but they do not propagate.  So we would have to add them for every dependency.

Reply:
Then I'm just confused about why we have both `hackage-quirks` and `configuration-nix` :thinking:

Reply:
We can't override the inputs to `cabal configure` from `configuration-nix`.